lib/src/module/built_in.dart

```diff
@@ -39,8 +41,16 @@ class BuiltInModule<T extends AsyncCallable> implements Module<T> {
           : UnmodifiableMapView(
               {for (var callable in callables) callable.name: callable}));
 
+  /// Returns a map from [variables]' names to their values.
+  static Map<String, V> _variableMap<V>(Map<String, V> variables) =>
```

Comment:
Since this logic isn't re-used, I don't think you need to factor it out into a function.

Reply:
Done.